app/src/androidTest/java/org/mozilla/focus/telemetry/TelemetryWrapperTest.java

```diff
@@ -96,17 +100,17 @@ public void launchByExternalAppEvent() {
     public void settingsEvent() {
         SharedPreferences pm = PreferenceManager.getDefaultSharedPreferences(InstrumentationRegistry.getTargetContext());
         for (String key : pm.getAll().keySet()) {
-            // TelemetryWrapper.settingsEvent(key, Boolean.FALSE.toString());
-            assertFirebaseEvent(TelemetryWrapper.Category.ACTION, TelemetryWrapper.Method.CHANGE, TelemetryWrapper.Object.SETTING, key, TelemetryWrapper.Extra.TO, Boolean.FALSE.toString());
+
+            TelemetryWrapper.settingsEvent(key, Boolean.FALSE.toString());
```

Comment:
The wrapper will skill non-whitelisted keys. So I've moved the assertion for whitelisted keys in a new test below.